pkg/descheduler/evictions/evictions.go

```diff
@@ -210,13 +596,39 @@ func evictPod(ctx context.Context, client clientset.Interface, pod *v1.Pod, poli
 		},
 		DeleteOptions: deleteOptions,
 	}
-	err := client.PolicyV1().Evictions(eviction.Namespace).Evict(ctx, eviction)
+	err := pe.client.PolicyV1().Evictions(eviction.Namespace).Evict(ctx, eviction)
+	if err == nil {
+		return false, nil
+	}
+	if pe.featureGates.Enabled(features.EvictionsInBackground) {
+		if pod.Annotations != nil {
+			// eviction in background requested
+			if _, exists := pod.Annotations[EvictionRequestAnnotationKey]; exists {
```

Comment:
we can simplify the annotations check also here

Reply:
Done